src/app/_components/CloudBackground.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+import Image from 'next/image';
+
+export default function CloudBackground({
+  children,
+}: {
+  children: React.ReactNode;
```

Comment:
```suggestion
  children: ReactNode;
```
💬 `import { ReactNode } from 'react';` 같은 식으로 구현하자고 했던 것 같습니다!